src/main/resources/templates/db_scripts/schema.json

```diff
@@ -85,6 +91,11 @@
       "run": "after",
       "snippetPath": "holdings_status_history_trigger.sql",
       "fromModuleVersion": "mod-kb-ebsco-java-3.0.2"
+    },
+    {
+      "run": "after",
+      "snippetPath": "access_types_update_metadata_user_info.sql",
+      "fromModuleVersion": "mod-kb-ebsco-java-3.0.1"
```

Comment:
why does version number differ from the one used with _"tableName": "access_types",_

Reply:
my bad, fixed